### ISSUE TITLE
Registrations CSV is named 'download.csv' instead of comp name

### ISSUE THREAD

**Issue by pranav-027** [thewca/worldcubeassociation.org#10292]
**Describe the bug**
In the new registration system, if you export the registration to a CSV file, it is named download.csv. The file was named after the competition in the old registration system, making it easier to distinguish in the file system.

**To Reproduce**
1. Go to the Registration page of any competition that is using the new reg system
2. Select all registrations 
3. click export to CSV

**Comment by FinnIckler** [comment]
we actually don't give a filename at all, so the file name is determined by the browser

**Comment by pranav-027** [comment]
Is it possible to implement something to make it mimic the functionality of v1 registration system

**Comment by FinnIckler** [comment]
yes, that was just an internal note to people who want to pick up the issue so they don't look for where 'download.csv' is defined
